app/webpacker/stylesheets/components/landing_pages/_support.scss

```diff
@@ -31,7 +31,7 @@
   box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+
```

Comment:
This class is across a few pages, have we checked to make sure this wasn't needed on one of the other pages?

Reply:
I've checked where we are using this class. It only appears on the I Belong page, Isaac Computer Science page and _support.html.erb partial which looks to be no longer used.

Doesn't effect the alignment on the Isaac page and fixes alignment on the I belong cards.